tests/Unit/IO/Observers/Test_ReductionObserver.cpp

```diff
@@ -102,20 +101,14 @@ SPECTRE_TEST_CASE("Unit.IO.Observers.ReductionObserver", "[Unit][Observers]") {
     file_system::rm(h5_file_name, true);
   }
 
-  using Redum = Parallel::ReductionDatum<double, funcl::Plus<>,
-                                         funcl::Sqrt<funcl::Divides<>>,
-                                         std::index_sequence<1>>;
-  using ReData = Parallel::ReductionData<
-      Parallel::ReductionDatum<double, funcl::AssertEqual<>>,
-      Parallel::ReductionDatum<size_t, funcl::Plus<>>, Redum, Redum>;
   const auto make_fake_reduction_data = [](
       const observers::ArrayComponentId& id, const double time) noexcept {
     const auto hashed_id =
         static_cast<double>(std::hash<observers::ArrayComponentId>{}(id));
     constexpr size_t number_of_grid_points = 4;
     const double error0 = 1.0e-10 * hashed_id + time;
     const double error1 = 1.0e-12 * hashed_id + 2 * time;
-    return ReData{time, number_of_grid_points, error0, error1};
+    return reduction_data{time, number_of_grid_points, error0, error1};
```

Comment:
Where is `reduction_data` defined?  I can't seem to find the definition.

Reply:
It's in `ObserverHelpers.hpp` (added in this same commit), being made available by the `using namespace TestObservers_detail` at the top of this file.

Reply:
Aah. I'm not a big fan of `using namespace...`, where the namespace is defined in a different file, for this very reason (it hurts readability). I would prefer writing out `TestObservers_detail::reduction_data` everywhere. But if everyone else is ok with `using namespace...` here, I won't complain anymore.

Reply:
Agreed, will change this.